lib/index.ts

```diff
@@ -20,19 +20,27 @@ const internals = {
 };
 
 function connect(token: string, options?: ConnectOptions): Promise<Room> {
-  return internals.connect(token, options);
+  const internalOptions = {
```

Comment:
So we pass in `createLocalTracks` here @charliesantos. Same for `createLocalVideoTrack` and `createLocalAudioTrack`.